R/pca.R

```diff
@@ -67,82 +67,87 @@ ui_g_pca <- function(id,
+              tags$label("Show Label"),
+              shinyWidgets::switchInput(ns("label"), value = TRUE, size = "mini")
+            ),
+            conditionalPanel(
+              condition = "input.tab_selected == 'PC and Sample Correlation'",
+              ns = ns,
+              tags$label("Cluster columns"),
+              shinyWidgets::switchInput(ns("cluster_columns"), value = FALSE, size = "mini")
+            ),
+            tags$label("View Matrix"),
+            shinyWidgets::switchInput(ns("show_matrix"), value = TRUE, size = "mini")
+          )
+        )
       ),
-      teal.widgets::panel_group(
-        teal.widgets::panel_item(
-          input_id = "settings_item",
-          collapsed = TRUE,
-          title = "Additional Settings",
-          tags$label("Use only Top Variance Genes"),
-          shinyWidgets::switchInput(ns("filter_top"), value = FALSE, size = "mini"),
-          conditionalPanel(
-            condition = "input.filter_top",
-            ns = ns,
-            sliderInput(ns("n_top"), label = "Number of Top Genes", min = 10, max = 5000, value = 500)
-          ),
-          conditionalPanel(
-            condition = "input.tab_selected == 'PCA'",
-            ns = ns,
-            tags$label("Show Variance %"),
-            shinyWidgets::switchInput(ns("var_pct"), value = TRUE, size = "mini"),
-            tags$label("Show Label"),
-            shinyWidgets::switchInput(ns("label"), value = TRUE, size = "mini")
+      output = div(
+        style = "display:flow-root",
+        tabsetPanel(
+          id = ns("tab_selected"),
+          type = "tabs",
+          tabPanel(
+            "PCA",
+            column(
+              width = 12,
+              div(
+                class = "my-5",
+                plotOutput(ns("plot_pca"))
+              ),
+              DT::DTOutput(ns("table_pca"))
+            )
           ),
-          conditionalPanel(
-            condition = "input.tab_selected == 'PC and Sample Correlation'",
-            ns = ns,
-            tags$label("Cluster columns"),
-            shinyWidgets::switchInput(ns("cluster_columns"), value = FALSE, size = "mini")
-          ),
-          tags$label("View Matrix"),
-          shinyWidgets::switchInput(ns("show_matrix"), value = TRUE, size = "mini")
-        )
-      )
-    ),
-    output = div(
-      style = "display:flow-root",
-      tabsetPanel(
-        id = ns("tab_selected"),
-        type = "tabs",
-        tabPanel(
-          "PCA",
-          column(
-            width = 12,
-            div(style = "height:20px;"),
-            plotOutput(ns("plot_pca")),
-            br(), br(), br(),
-            DT::DTOutput(ns("table_pca"))
-          )
-        ),
-        tabPanel(
-          "PC and Sample Correlation",
-          column(
-            width = 12,
-            div(style = "height:20px;"),
-            plotOutput(ns("plot_cor")),
-            br(), br(), br(),
```

Comment:
these not needed any more

Reply:
what you mean

Reply:
The `br(), br(), br()` is missing now?
![image](https://user-images.githubusercontent.com/15201933/181755330-b9cfb86c-263d-4bf5-8f38-cc0ac481622a.png)

Is that OK?

Reply:
my-5 replace the upper div and the lowe br's. It is ok.